model/variableSheet.h

```diff
@@ -0,0 +1,43 @@
+/*
+  @copyright Steve Keen 2018
+  @author Russell Standish
+  This file is part of Minsky.
+
+  Minsky is free software: you can redistribute it and/or modify it
+  under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  Minsky is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with Minsky.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/**
+   Adds a tab to eventually edit variables in one place
+*/
+
+#ifndef VARIABLESHEET_H
+#define VARIABLESHEET_H
+#include <cairoSurfaceImage.h>
+
+namespace minsky
+{
+	 
+  class VariableSheet: public ecolab::CairoSurface
+  {
+  public:
+    double xoffs=80;  
+    double rowHeight=0;
+    double colWidth=50;  
+    void redraw(int, int, int width, int height) override;
+    void requestRedraw() {if (surface.get()) surface->requestRedraw();}
```

Comment:
Maybe this bog standard requestRedraw (which exposes this functionality to TCL) should be provided by ecolab::CairoSurface. It can be overridden when needed (PlotWidget needs to do something different)

Reply:
We use  ecolab::CairoSurface in redraw() so it makes sense to use it in requestRedraw() too. Okay, understood.